### PR TITLE
Update Interactable docs with toggle and dimensions clarification

### PR DESCRIPTION
**Pull request by killerantz**
## Overview
A question about how the check box example in the InteractiveExamples scene. #4009 
How to trigger the toggled state.

Added some details to the Toggles section.

### CODE REVIEW

**Comment by killerantz** [comment]
I redirected this PR to MRTK_developement instead of MRTK_release.

**Comment by julenka** [comment]
@killerantz  It looks like there is a broken image, could you please fix? Other than that looks good to me and I'll merge it in.

image

File: Documentation/README_Interactable.md [hunk 1]
```diff
@@ -55,41 +55,64 @@ Themes work a lot like materials. They are scriptable objects that contain a lis
...
+SetDimensionIndex(1);
+
+//Set the DimensinIndex - Untoggled
+SetDimensionIndex(0);
+```
 
-You can use an interactable to detect input events other than just OnClick. The events feature provides a way to enable functionality to extend a button, but not really visual or needed to provide feedback based on state changes.
+Every click event will advance the DimensionIndex which will increase until the set Dimensions value is reached then cycle or reset to 0. A good example of working with Dimensions with code is the InteractiveToggleCollection found in the InteractableExamples demo scene on the RadialSet object.
+
+<img src="../Documentation/ReadMeImages/Interactable/InteractableToggleCollection.png" width="450">
```
Comment (keveleigh) [inline Documentation/README_Interactable.md]:
```suggestion
<img src="../Documentation/Images/Interactable/InteractableToggleCollection.png" width="450">
```

Comment (killerantz) [reply Documentation/README_Interactable.md]: Thanks for that. Totally thought I tested the link.

**Pull request merged**